### ISSUE TITLE
Wide review request for Pointer Events Level 3

### ISSUE THREAD

**Issue by patrickhlauke** [w3ctag/design-reviews#941]
こんにちは TAG-さん!

I'm requesting a TAG review of the proposed Pointer Events Level 3 specification.

> This specification is an update to [[PointerEvents2](https://www.w3.org/TR/2024/WD-pointerevents3-20240318/#bib-pointerevents2)] which was shipped broadly by Google Chrome and Microsoft Edge and Mozilla Firefox. Level 3 includes editorial clarifications and new features that facilitate more use cases, in an effort to enable wider developer and browser adoption.

  - Explainer¹ (minimally containing user needs and example code): https://www.w3.org/TR/2024/WD-pointerevents3-20240318/#intro
  - Specification URL: https://www.w3.org/TR/2024/WD-pointerevents3-20240318/
  - Tests: https://wpt.fyi/pointerevents/ (though some are still being refined/worked on)
  - Security and Privacy self-review²: https://www.w3.org/TR/2024/WD-pointerevents3-20240318/#security-and-privacy-considerations
  - GitHub repo (if you prefer feedback filed there): https://github.com/w3c/pointerevents/
  - Primary contacts (and their relationship to the specification):
      - Patrick H. Lauke ([patrickhlauke](https://github.com/patrickhlauke), TetraLogical - editor and WG chair)
      - Robert Flack ([flackr](https://github.com/flackr), Google - editor)
  - Organization(s)/project(s) driving the specification: Google, Mozilla
  - Key pieces of existing multi-stakeholder (e.g. developers, implementers, civil society) support, review or discussion of this specification:
    - Chromium comments: [url]
    - Mozilla comments: 
    - WebKit comments: 
    - Etc.
  - External status/issue trackers for this specification (publicly visible, e.g. Chrome Status):

Further details:

  - [x] I have reviewed the TAG's [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
  - Relevant time constraints or deadlines: we would welcome feedback before 12 June 2024
  - The group where the work on this specification is currently being done: PEWG https://www.w3.org/groups/wg/pointer-events/
  - The group where standardization of this work is intended to be done (if current group is a community group or other incubation venue):
  - Major unresolved issues with or opposition to this specification: None
  - This work is being funded by:

You should also know that...

[please tell us anything you think is relevant to this review]

We'd prefer the TAG provide feedback as (please delete all but the desired option):

  ☂️ open a single issue in our GitHub repo **for the entire review**

**Comment by torgo** [comment]
Hi @patrickhlauke  thanks for bringing this to us. We're closing this as `satisfied`. We're happy to see this work continue to move forward...

**Comment by patrickhlauke** [comment]
Thank you @torgo and TAG
